### PR TITLE
docs(clean-up): translate to zh-cn

### PR DESCRIPTION
**Pull request by ishowman**

### CODE REVIEW

**Review by Mireille-T — CHANGES_REQUESTED** [review]
Thanks again @ishowman for the translation! 😄 

May I request the following changes?
- `一个自定义的愿景项目` to `一个自定义视觉项目` in line 13; I think it may be better to follow the Microsoft documentation for certain technical terms, such as "[Custom Vision](https://docs.microsoft.com/zh-cn/azure/cognitive-services/custom-vision-service/)" in this case.
- I feel `这些资源中大部分都不收费 - 完全免费或者是免费套餐` from line 17 can be translated better. May I suggest `这些资源大多数不收费 - 有的完全免费，或者你正在用免费层`; `免费层` is once again taken from Microsoft Docs, so I think the translation of "tier" to "层" can be used (for the rest of the file as well).
- In line 29, the phrase "interested in" from the English translation does not literally mean interest / passion, so perhaps instead of `你感兴趣的资源组`, we can use `你想删除的资源组` i.e. the resource group that you want to delete.
- In line 41, to keep in line with the English version, maybe we can change `Azure 资源管理器资源组和资源删除` to `Microsoft Docs 的 Azure 资源管理器资源组和资源删除文档`

Do let me know if you think any of the above suggestions can be improved upon.

**Comment by ishowman** [comment]
> Thanks again @ishowman for the translation! 😄
> 
> May I request the following changes?
> 
> * `一个自定义的愿景项目` to `一个自定义视觉项目` in line 13; I think it may be better to follow the Microsoft documentation for certain technical terms, such as "[Custom Vision](https://docs.microsoft.com/zh-cn/azure/cognitive-services/custom-vision-service/)" in this case.
> * I feel `这些资源中大部分都不收费 - 完全免费或者是免费套餐` from line 17 can be translated better. May I suggest `这些资源大多数不收费 - 有的完全免费，或者你正在用免费层`; `免费层` is once again taken from Microsoft Docs, so I think the translation of "tier" to "层" can be used (for the rest of the file as well).
> * In line 29, the phrase "interested in" from the English translation does not literally mean interest / passion, so perhaps instead of `你感兴趣的资源组`, we can use `你想删除的资源组` i.e. the resource group that you want to delete.
> * In line 41, to keep in line with the English version, maybe we can change `Azure 资源管理器资源组和资源删除` to `Microsoft Docs 的 Azure 资源管理器资源组和资源删除文档`
> 
> Do let me know if you think any of the above suggestions can be improved upon.

Thanks for review.
I will make all the modification today later.

File: translations/clean-up.zh-cn.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+* 一个物联网中心
+* 物联网设备注册
+* 一个存储账户
+* 一个函数应用
+* 一个 Azure 地图账户
+* 一个自定义视觉项目
+* 一个 Azure 容器注册表
+* 一个认知服务资源
+
+这些资源中大部分都不收费 - 完全免费或者是免费层。对于规定了付费层的服务，你的使用额度会在免费范围内，或只花费几分钱。
```
Comment (ishowman) [inline translations/clean-up.zh-cn.md:17]: @Mireille-T  I've done.May you review it again later? 
And you can add comment to anywhere you request for modifing like this by clicking the line number in `Files changed` tab.

File: translations/clean-up.zh-cn.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+即使成本相对较低，当你完成项目后也建议删除这些资源。例如，免费层只能有一个物联网中心，所以如果你想额外创建一个，你需要使用付费套餐。
```
Comment (Mireille-T) [inline translations/clean-up.zh-cn.md:19]: I think `付费套餐` here still needs to be changed to `付费层`. 😅

File: translations/clean-up.zh-cn.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```output
+Are you sure you want to perform this operation? (y/n): 
+```
+
+输入 `y` 确认并删除资源组.
+
+删除所有服务会耗费一些时间。
+
+> 💁 你可以在 [Microsoft Docs 的 Azure 资源管理器资源组和资源删除](https://docs.microsoft.com/azure/azure-resource-manager/management/delete-resource-group?WT.mc_id=academic-17441-jabenn&tabs=azure-cli) 获取更多删除资源组的相关内容。
```
Comment (Mireille-T) [inline translations/clean-up.zh-cn.md:41]: Do you think we should add `文档` to the back of `Azure 资源管理器资源组和资源删除` as a translation for "documentation"?

**Review by Mireille-T — CHANGES_REQUESTED** [review]
Thanks again @ishowman, just a couple more changes requested. I've indicated the comments below the lines, thank you for letting me know about that. 😅

File: translations/clean-up.zh-cn.md (hunk 2 above)
Comment (ishowman) [reply translations/clean-up.zh-cn.md]: done.

File: translations/clean-up.zh-cn.md (hunk 3 above)
Comment (ishowman) [reply translations/clean-up.zh-cn.md]: done.

**Review by Mireille-T — APPROVED** [review]
Alright, @jimbobbennett I think this PR can be merged now! Thank you once again @ishowman! :smile:

**Comment by ishowman** [comment]
> Alright, @jimbobbennett I think this PR can be merged now! Thank you once again @ishowman! 😄

It's my pleasure.By the way, would you mind telling me what country you are living in, America?

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**